src/main/msp/msp.c

```diff
@@ -2839,7 +2848,6 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, uint8_t cmdMSP,
         rxConfigMutable()->maxcheck = sbufReadU16(src);
         rxConfigMutable()->midrc = sbufReadU16(src);
         rxConfigMutable()->mincheck = sbufReadU16(src);
-        rxConfigMutable()->spektrum_sat_bind = sbufReadU8(src);
```

Comment:
Why would you remove this? Doing so will corrupt the byte order of the message contents.

Reply:
Fat fingers. Now reverted.

Reply:
This was the cause of my bug, sorted now 👍 .